app/_src/kubernetes-ingress-controller/reference/combined-services-from-different-httproutes.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Combined Services From Different HTTPRoutes 
+type: reference
+purpose: |
+  Introduce the feature gate to consolidate {{site.base_gateway}} services by combining rules from different HTTPRoutes
+alpha: true
+---
+
+As have been done to `Ingress`es, {{site.kic_product_name}} can consolidate rules from different `HTTPRoute`s with the same
+combination of backend services and translate them into one {{site.base_gateway}} service to reduce the number of {{site.base_gateway}} services.
+The feature is enabled when the feature gate `CombinedServicesFromDifferentHTTPRoutes` is set to `true`.
+When the feature is enabled, The rules with the same combination of backend services(combination of namespace, name, port and weight in `backendRefs` of rules)
+in all `HTTPRoute`s within the same namespace will be translated to one {{site.base_gateway}} service.
+
+## Changes on Translated {{site.base_gateway}} Service Names
+
+The names of translated {{site.base_gateway}} service will be changed when the feature is enabled. Instead of generating names from source `HTTPRoute`
```

Comment:
```suggestion
The names of the translated {{site.base_gateway}} service will be changed when the feature is enabled. Instead of generating names from source `HTTPRoute`
```